docs/extend/extend-apiml/api-mediation-infinispan.md

```diff
@@ -31,15 +31,14 @@ Configure Infinispan as a storage solution through the Caching service by settin
   This property specifies the list of cluster nodes (members). In case of multiple instances, the value for each Caching Service instance can be 
   either a list of all the members, separated by a comma, or just the replica. The format is `${haInstance.hostname}[${zowe.components.caching-service.storage.infinispan.jgroups.port}]`.
 
-
 * **`zowe.components.caching-service.storage.infinispan.persistence.dataLocation`**
 
-  The path where the Soft-Index store will keep its data files for the Infinispan Soft-Index Cache Store. 
+  The path where the Soft-Index store keeps its data files for the Infinispan Soft-Index Cache Store. 
   The default value is `data`. If you run the Caching Service in HA and the instances use the same filesystem,
+
   you have to specify a different value of the `zowe.components.caching-service.storage.infinispan.persistence.dataLocation` property for each
```

Comment:
"... you must specify a different value ..."

Reply:
I made a mistake when I created this branch and, as a result, it now includes another writer's content. Since this is the work of another writer, I will not be editing this file.